### PR TITLE
Fix animation issue with iPhone 5/SE

### PR DESCRIPTION
**Pull request by jamieliu386**
Fixes: #13

### CODE REVIEW

File: src/components/HomePage/FAQSection.js [hunk 1]
```diff
@@ -4,49 +4,58 @@ import { Typography, Link } from '@material-ui/core';
...
 import Add from '@material-ui/icons/Add';
 import Remove from '@material-ui/icons/Remove';
 
 const ExpansionPanel = withStyles(theme => ({
 	root: {
 		backgroundColor: theme.palette.background.default,
-		borderTop: '1px solid rgba(0, 0, 0, 1)',
-		borderBottom: '1px solid rgba(0, 0, 0, 1)',
+		borderTop: '1px solid black',
+		borderBottom: '1px solid black',
```
Comment (TimothyGu) [inline src/components/HomePage/FAQSection.js:16]: Black is the same thing as the above, only better known.

File: src/components/HomePage/FAQSection.js [hunk 2]
```diff
@@ -4,49 +4,58 @@ import { Typography, Link } from '@material-ui/core';
...
 		boxShadow: 'none',
 		'&:not(:last-child)': {
 			borderBottom: 0
 		},
-		'&:before': {
+		'&::before': {
```
Comment (TimothyGu) [inline src/components/HomePage/FAQSection.js:21]: `::before` is a [pseudo-element](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements) which ordinarily are prefixed by `::`. `:before` is a compatibility syntax, as a single `:` refers to [pseudo-class](https://developer.mozilla.org/en-US/docs/Web/CSS/pseudo-classes).

File: src/components/HomePage/FAQSection.js [hunk 3]
```diff
@@ -4,49 +4,58 @@ import { Typography, Link } from '@material-ui/core';
...
 			display: 'none'
 		},
 		'&$expanded': {
-			margin: 'auto'
+			margin: 0
```
Comment (TimothyGu) [inline src/components/HomePage/FAQSection.js:25]: In this context, both are equivalent – but 0 makes a bit more sense.

File: src/components/HomePage/FAQSection.js [hunk 4]
```diff
@@ -4,49 +4,58 @@ import { Typography, Link } from '@material-ui/core';
...
 	root: {
-		minHeight: 20,
+		minHeight: 'auto',
 		'&$expanded': {
-			minHeight: 20
+			minHeight: 'auto'
+		},
+		[theme.breakpoints.down('sm')]: {
+			paddingLeft: 8,
+			paddingRight: 8
```
Comment (TimothyGu) [inline src/components/HomePage/FAQSection.js:39]: As discussed, this makes the page take up more space on the smallest devices that lack much display real estate. `ExpansionPanelDetails` is modified to match this.

File: src/components/HomePage/FAQSection.js [hunk 5]
```diff
@@ -4,49 +4,58 @@ import { Typography, Link } from '@material-ui/core';
...
-			minHeight: 20
+			minHeight: 'auto'
+		},
+		[theme.breakpoints.down('xs')]: {
+			paddingLeft: 8,
+			paddingRight: 8
 		}
 	},
 	content: {
+		margin: '12px 0',
```
Comment (TimothyGu) [inline src/components/HomePage/FAQSection.js:43]: 12px is chosen as that's what the framework does internally. 0 means no horizontal margin, which is also what the framework does (and which is all we need).

File: src/components/HomePage/FAQSection.js (hunk 2 above)
Comment (jamieliu386) [reply src/components/HomePage/FAQSection.js:21]: Oops, thanks for pointing this out!

**Comment by jamieliu386** [comment]
Thanks for making these edits, Tim!

**Pull request merged**